lms/djangoapps/grades/events.py

```diff
@@ -0,0 +1,134 @@
+from crum import get_current_user
```

Comment:
**Note to reviewers:** I added this new file to encapsulate all eventing related code.  This helps improve the readability of the code elsewhere and also helps see at-a-glance the grading-related events that are generated.

Reply:
This is great. So much easier to maintain!